hastexo_guacamole_client/consumers.py

```diff
@@ -66,7 +66,7 @@ async def disconnect(self, code):
         Close the GuacamoleClient connection on WebSocket disconnect.
         """
         self.task.cancel()
-        await sync_to_async(self.client.close)()
+        await sync_to_async(self.client.close, thread_sensitive=False)()
```

Comment:
I suggest adding a comment here to document why we want this to be `False` and not the default `True`.

Reply:
Ack, done.